tooltester/README.md

```diff
@@ -47,65 +80,84 @@ Now, run
 agc context describe <CONTEXT NAME>
 ```
 on `wdlContext` and `cwlContext`, and get the `WESENDPOINT` value from the result and append `ga4gh/wes/v1` to the end of it and place
-the resulting string in the `url` field in `WDLConfigFile` and `CWLConfigFile` respectively.
+the resulting string in the `WDL-WES-URL` and `CWL-WES-URL` field in `tooltesterConfig.yml` respectively.
 
-Now, you will want to add your AWS profile name to both `wdlContext` and `cwlContext` in the `authorization` field. For example,
+NOTE: All contexts on AGC are user-scoped, so having a context with the same name as a context created by someone else is not a problem.
+
+Now, you will want to add your AWS profile name to the `AWS-AUTHORIZATION` field in `tooltesterConfig.yml`. For example,
 mine is `fhembroff`.
 
-### Finish writing Config Files
-Do the following to both `WDLConfigFile` and `CWLConfigFile`.
+### Add Server Url and Token to `tooltesterConfig.yml`
+
+In this step you must fill in the `TOKEN` and `SERVER-URL` in `tooltesterConfig.yml`. The `SERVER-URL` will most likely
+either be https://qa.dockstore.org or https://dockstore.org. The token is obtained from whatever dockstore site you chose
```

Comment:
Urls should have `/api` at the end of them?

Reply:
Good catch, I have fixed this.